app/population/controllers/edit.js

```diff
@@ -50,6 +51,14 @@ class DetailsController extends FormWizardController {
     }
   }
 
+  setBreadcrumbs(req, res, next) {
+    res.breadcrumb({
```

Comment:
Why put it on `res` rather than `req`? (I believe the "pattern" we're following is stash everything on `req` and then update `res.locals` prior to rendering)

Reply:
@solidgoldpig I've updated the ticket with a little more context - basically this is slightly tweaked reusing of an unmerged PR of @teneightfive 's

It sounds like our development patterns have changed in that time, so refactoring this to match the other code is a good idea

Reply:
The reason for this being on `res` instead of `req` is because it is concerned with the response rather than the original request.

The major benefit is that it can be chained with other `res` methods like `res.render`.

For example:

```
res
  .breadcrumb(...)
  .render(...)

```

Reply:
It's also a different use case to the previous things we have attached to the request object.

Reply:
I hadn't seen this followup discussion, and in the meantime I'd switched most usages to run off `req` as per my discussion with Alex. It's only a single commit, so can be easily dropped.

Reply:
I think it would be good to keep the original implementation on `res`. It serves a different purpose to how we've attached things to the `req`.

The breadcrumb feels more part of the response that the request too and I think the benefit of being able to chain `breadcrumb()` and `render()` together is a nice pattern.

Reply:
I've dropped the commit that used `req` and left it as per the original implementation.